.github/workflows/deployment.yml

```diff
@@ -40,7 +40,7 @@ on:
 env:
   PRODUCT: apim
   ENVIRONMENT: ${{ github.ref_name }}
-  TIMEZONE: "Europe/London"
+  timezone: ${{ vars.TIMEZONE }}
```

Comment:
Should we keep upper case env var name to be consistent?

Reply:
Thanks, have converted to lower case to keep it consistent with other projects will make them all uppercase in a separate PR.